WPThemeReview/Tests/CoreFunctionality/PrefixAllGlobalsUnitTest.php

```diff
@@ -0,0 +1,75 @@
+<?php
+/**
+ * Unit test class for WPThemeReview Coding Standard.
+ *
+ * @package WPTRT\WPThemeReview
+ * @link    https://github.com/WPTRT/WPThemeReview
+ * @license https://opensource.org/licenses/MIT MIT
+ */
+
+namespace WPThemeReview\Tests\CoreFunctionality;
+
+use PHP_CodeSniffer\Tests\Standards\AbstractSniffUnitTest;
+
+/**
+ * Unit test class for the PrefixAllGlobals sniff.
+ *
+ * @package WPTRT\WPThemeReview
+ *
+ * @since 0.2.0
+ */
+class PrefixAllGlobalsUnitTest extends AbstractSniffUnitTest {
+
+	/**
+	 * Returns the lines where errors should occur.
+	 *
+	 * @param string $testFile The name of the file being tested.
+	 * @return array <int line number> => <int number of errors>
+	 */
+	public function getErrorList( $testFile = 'partials/post-edit.inc' ) {
+		switch ( $testFile ) {
+			case 'social-share.inc':
+				return array(
+					6 => 1,
+				);
+			case 'header.inc':
+				// Template file - all OK, fall through to the default case.
+			case 'footer_widgets.inc':
+				return array(
+					6 => 1,
+				);
+			case 'post-edit.inc':
+				// Template file - all OK, fall through to the default case.
+			default:
+				return array();
+		}
+	}
+
+	/**
```

Comment:
As a convention, this method should be above the `getErrorList()`/`getWarningList()` methods (as it will be called before either of them).

Reply:
Yeah, this makes more sense 🙂